### PR TITLE
Implement the payments database boundaries and functions

### LINKED ISSUE #93: Implement the payments database boundaries and functions

### PR DESCRIPTION
**Pull request by jhonatan-kmt**
## Description

Create the payments schema and database functions 

Closes #93 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by SanGeeky — CHANGES_REQUESTED** [review]

File: lib/mintacoin/payments/payment.ex [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    field(:status, Status, default: :processing)
+    field(:successful, :boolean)
+
+    timestamps()
+  end
+
+  @spec changeset(payment :: %__MODULE__{}, changes :: map()) :: Changeset.t()
+  def changeset(payment, changes) do
+    payment
+    |> cast(changes, [:status])
```
[the remark this answers is not in the document]
Comment (SanGeeky) [reply lib/mintacoin/payments/payment.ex:49]: I think is also necessary, the field `:successful`,  in case of a failed transaction.

**Review by SanGeeky — COMMENTED** [review]
LGTM :sunrise_over_mountains:

File: lib/mintacoin/payments/payment.ex (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhonatan-kmt) [reply lib/mintacoin/payments/payment.ex:49]: Ok, we will add it

**Review by SanGeeky — APPROVED** [review]
:rocket:

**Pull request merged**